package.json

```diff
@@ -37,15 +37,6 @@
   },
   "devDependencies": {
     "concurrently": "^5.3.0",
-    "eslint": "^7.7.0",
```

Comment:
we need eslint to keep the code clean. you can add the config in .env file. SKIP_PREFLIGHT_CHECK=true